lib/mockbuilder.py

```diff
@@ -17,6 +17,7 @@
 
```

Comment:
OH> I don't get your second point, doesn't your suggestion of using a primary key lead to package (and commit) specific directories?

I was under the impression in my first quick read of the code that, if you are building A and B and C are dependencies, you'd put B and C rpms inside A dir. In that case, if C was a dependency for D, you would not find C inside D dir and rebuild C.